Current-Chapters.csv

```diff
@@ -108,7 +108,8 @@
 "Luxembourg","","Luxembourg","","","","","","","","","","","Claudia Albanese","Prospective"
 "Mexico","","Mexico City","https://www.meetup.com/rladies-cdmx/","https://twitter.com/RLadiesCDMX","cdmx@rladies.org","https://www.facebook.com/rladiescdmx","","","","","","rladiescdmx.slack.com","Silvia Eunice Gutiérrez de la Torre, Maria Ortiz, Mariana Carmona, Mariana Godina","Active"
 "Mexico","","Morelia","","","","","","","","","","","Thomai Tsiftsi","Prospective"
-"Mexico","","Puebla","","","puebla@rladies.org","","","","","","","","Averi Giudicessi","Prospective"
+yabellini-patch-42
```

Comment:
@yabellini - Not sure why it is showing `yabellini-patch-42`

Reply:
Do I have to do something??

Reply:
I deleted that line. Now don't show any conflict.